dev/conductor/core/test/start_test.dart

```diff
@@ -76,6 +76,86 @@ void main() {
       return CommandRunner<void>('codesign-test', '')..addCommand(command);
     }
 
+    group('start arg parser', () {
+      const String nextDartRevision =
+          'f6c91128be6b77aef8351e1e3a9d07c85bc2e46e';
+      late StartCommand startCommand;
+      setUp(() {
+        String operatingSystem = const LocalPlatform().operatingSystem;
+        final String pathSeparator = operatingSystem == 'windows' ? r'\' : '/';
```

Comment:
To simplify things, we just dis-allow Windows: https://github.com/flutter/flutter/blob/master/dev/conductor/core/lib/src/start.dart#L120

Reply:
Removed. This just came from the existing create runner code. https://github.com/flutter/flutter/blob/master/dev/conductor/core/test/start_test.dart#L50